### PR TITLE
fix(benchpress): explicitly require navigation to finish before continui...

### PR DESCRIPTION
**Pull request by juliemr**
...ng

### CODE REVIEW

File: modules/angular2/src/test_lib/perf_util.es6 [hunk 1]
```diff
@@ -33,14 +33,16 @@ function runBenchmark(config) {
     }
     var url = encodeURI(config.url + '?' + urlParams.join('&'));
     browser.get(url);
```
Comment (gkalpak) [inline modules/angular2/src/test_lib/perf_util.es6]: Shouldn't this line be deleted as well ?

**Comment by tbosch** [comment]
Agree with @gkalpak. Otherwise LGTM.

File: modules/angular2/src/test_lib/perf_util.es6 (hunk 1 above)
Comment (juliemr) [inline modules/angular2/src/test_lib/perf_util.es6]: Yup, done.

**Comment by juliemr** [comment]
Merged as 8b28e99373d3032eef89cad97a87d7d594037a10

**Pull request closed without merging**